c7n/resources/ses.py

```diff
@@ -336,3 +340,189 @@ def process(self, rulesets):
                 RuleSetName=ruleset["Metadata"]['Name'],
                 ignore_err_codes=("CannotDeleteException",)
             )
+
+
+class DescribeSESIngressEndpoint(DescribeSource):
+
+    def augment(self, resources):
+        client = local_session(self.manager.session_factory).client('mailmanager')
+
+        def _augment(r):
+            tags = client.list_tags_for_resource(ResourceArn=r['IngressPointArn'])['Tags']
+            r['Tags'] = tags
+            return r
+        resources = super().augment(resources)
+        return list(map(_augment, resources))
```

Comment:
While it looks like we can't use the Resource Groups Tagging API to _set_ tags, it does look like we can use it to _fetch_ them. So we should be able to use a `universal_augment()` here or use the [DescribeWithResourceTags](https://github.com/cloud-custodian/cloud-custodian/blob/148dfe1c1b2169dc989990b7481fb15b572e8f60/c7n/query.py#L280-L283) source ([example](https://github.com/cloud-custodian/cloud-custodian/blob/148dfe1c1b2169dc989990b7481fb15b572e8f60/c7n/resources/appstream.py#L19-L21)).

Reply:
I didn't know that before. Thanks!